content/courses/mobile/solana-mobile-dapps-with-expo.md

```diff
@@ -260,8 +270,8 @@ Let's practice this together by building the Mint-A-Day app, where users will
 able to mint a single NFT snapshot of their lives daily, creating a permanent
 diary of sorts.
 
-To mint the NFTs we'll be using Metaplex's Javascript SDK along with
-[nft.storage](https://nft.storage/) to store images and metadata. All of our
+To mint the NFTs we'll be using Metaplex's Umi libraries along with
+[Pinata Cloud](https://pinata.cloud/) to store images and metadata. All of our
 onchain work will be on Devnet.
```

Comment:
Pinata is fine, but please add a ContentBox saying something like "We are using Pinata in this tutorial, but [there are many good solutions for store images for long-term storage](https://solana.com/developers/guides/getstarted/how-to-create-a-token#create-and-upload-image-and-offchain-metadata)"

Foundation can't favor particular tech, so we need to add this!